flang/include/flang/Lower/FIRBuilder.h

```diff
@@ -257,8 +257,12 @@ class FirOpBuilder : public mlir::OpBuilder,
 
   const fir::KindMapping &getKindMap() { return kindMap; }
 
-  mlir::Value convertOnAssign(mlir::Location loc, mlir::Type toTy,
-                              mlir::Value val);
+  /// The LHS and RHS are not always in agreement in terms of
```

Comment:
I think I heard that Tim is fixing this in the front end, he reported yesterday.

Reply:
f18-stage | Make implicit conversion explicit in assignment
Merge Request !23 was merged

Do you want me to ask Tim to push this to llvm-project?